plugins/CorePluginsAdmin/PluginInstaller.php

```diff
@@ -49,6 +50,11 @@ public function installOrUpdatePluginFromMarketplace()
 
             Filesystem::deleteAllCacheOnUpdate($this->pluginName);
 
+            $plugin = PluginManager::getInstance()->getLoadedPlugin($this->pluginName);
+            if (!empty($plugin)) {
+                $plugin->reloadPluginInformation();
```

Comment:
Here we need to make sure to reload the plugin json which contains the required piwik version etc. Otherwise plugin would be directly disabled after the update because it still holds the old update information